TGEngine/gamecontent/camera/StaticCamera.hpp

```diff
@@ -6,10 +6,26 @@
 namespace tge {
 	namespace gmc {
 
+		/*
+		 * This creates a camera that is static
+		 * but the rendered objects can be turned
```

Comment:
```suggestion
		 * This creates a static camera
		 * but the rendered objects can be turned
```
what exactly do you mean with "but the rendered objects can be turned"? do you mean rotated? please clarify